CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-### 2.0.3 (Next)
+### 2.1.0 (Next)
 
+* [#44](https://github.com/acm19/aws-request-signing-apache-interceptor/issues/44): Release to Maven Central - [@dblock](https://github.com/dblock),[@acm19](https://github.com/acm19).
```

Comment:
Add a space after the `,` in `,[@acm19]()`

Reply:
Done.